Integration/bootstrap.php

```diff
@@ -0,0 +1,124 @@
+<?php
```

Comment:
Do we want to add all of this to the package?

Reply:
All packages should have integration tests that need bootstrapping. Why?

- The common fixtures are loaded in this bootstrapper. 
- We can encapsulate certain constants that we use everywhere such as `WP_ROCKET_IS_TESTING`.

For tasks that are loaded, many are guarded by which composer script we're running. Encapsulating them here makes them available everywhere, even when a module doesn't need it. That centralizes the bootstrapping. Use what you needed.

Reply:
I'm thinking specifically about all the groups & WooCommerce installation indeed, where I'm approaching this package to make it as light as possible, and add specificity when needed only

Reply:
But it needs to be inclusive. The groups will not do anything unless they are run. Having them here ensures they are available when needed.

Now for the installations, we could target those. For example, we could remove WooCommerce for now and add it back if it's use case grows to more modules.

Reply:
Removed the WC composer dep and the integration theme loader. But kept the group as it won't do anything unless that group is called. See commit https://github.com/wp-media/phpunit-wp-rocket/pull/1/commits/2a4e7b85d009fbdd17de30e87399c65def19fe69